playbooks/check-versions.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Fail if Ansible version is too old
+      fail:
+        msg: "Installed Ansible version is too old. Upgrade to at least 2.12.7"
+      when: (ansible_version.major < 2) or
+            (ansible_version.major == 2 and ansible_version.minor < 12) or
+            (ansible_version.major == 2 and ansible_version.minor == 12 and ansible_version.revision < 7)
+    - name: Check Jinja2 version
+      shell: | 
+        JINJA_VERSION=$(pip3 freeze | grep Jinja2 | sed 's/Jinja2==//g')
+        echo $JINJA_VERSION | tr '.' '\n'
+      register: version
```

Comment:
use something more specific like jinja2_version because in ansible all variables are defined in a shared global context and can save a lot of headaches